installation-manager-core/src/main/java/com/codenvy/im/report/ReportType.java

```diff
@@ -32,8 +32,7 @@ public ReportParameters getParameters() {
         ReportParameters parameters = new ReportParameters(
```

Comment:
I would stay it as it to make it possible to change report parameters in config file without obligation to restart _update server_.